### PR TITLE
Switch to use CodeStructureService in DocumentSymbolHandler.

### LINKED ISSUE #1407: [LSP] DocumentSymbolHandler should use CodeStructureService
DocumentSymbolHandler should use CodeStructureService and return DocumentSymbols to support Outline view in VSCode.

Currently it uses old MembersAsTreeService.

### PR DESCRIPTION
**Pull request by mholo65**
This PR will get outline view working in VS Code when using LSP.
image

Needs https://github.com/OmniSharp/csharp-language-server-protocol/pull/123 (ping @david-driscoll)

Fixes #1407

### CODE REVIEW

File: build/Packages.props [hunk 1]
```diff
@@ -60,7 +60,7 @@
     <PackageReference Update="Nuget.ProjectModel" Version="$(NuGetPackageVersion)" />
     <PackageReference Update="Nuget.Versioning" Version="$(NuGetPackageVersion)" />
 
-    <PackageReference Update="OmniSharp.Extensions.LanguageServer" Version="0.11.3" />
+    <PackageReference Update="OmniSharp.Extensions.LanguageServer" Version="0.11.4-alpha0011" />
```
Comment (mholo65) [inline build/Packages.props]: Needs stable release.

Comment (rchande) [reply build/Packages.props]: What is this package?

Comment (mholo65) [reply build/Packages.props]: @rchande it's the LSP bits from [here](https://github.com/OmniSharp/csharp-language-server-protocol)

**Comment by mholo65** [comment]
Works (but still blocked by https://github.com/OmniSharp/csharp-language-server-protocol/pull/129)
image

**Comment by mholo65** [comment]
@david-driscoll don't know if LSP update broke renamehandler, but at least it's not working. Seems to pick the wrong symbol.

image


image

**Comment by mholo65** [comment]
Updated LSP packages to release. Fix rename stuff in separate PR here #1423 

Ready for review @david-driscoll @filipw @rchande

**Pull request merged**